README.md

```diff
@@ -1,6 +1,15 @@
 # Aquavolution
 Aquavolution is a browser-based survival/adventure 2D game made with Unity.
 
+# Inspiration 
+# What it does 
+# How we built it(Each member talks about their additions) 
+# Challenges we ran into (i.e. working w/cinemachine, choosing the best algorithm to fit our needs of creating levels)
```

Comment:
delete (i.e. working w/cinemachine, choosing the best algorithm to fit our needs of creating levels) ...........................................................................